sycl/include/sycl/ext/oneapi/matrix/matrix-unified.hpp

```diff
@@ -25,6 +25,7 @@
 #include <sycl/ext/oneapi/matrix/query-types.hpp> // for convertTypeToMatrixTypeString
 #include <sycl/marray.hpp>                        // for marray
 #include <sycl/multi_ptr.hpp>                     // for multi_ptr
+#include <sycl/ext/oneapi/bfloat16.hpp> // bfloat16
```

Comment:
We should not add an another header dependency here so one option is to move the copy body to matrix-unified-utils.hpp